### PR TITLE
Adding R&S VISA

### PR DESCRIPTION
**Pull request by casabre**
Added R&S Visa to the library dependency aliases and to the documentation

### CODE REVIEW

File: doc/installation.rst [hunk 1]
```diff
@@ -18,6 +18,8 @@ this is available for all platforms.
 * Linux (32 bit only [#f1]_ ): `NI-VISA 14.0 <http://www.ni.com/download/ni-visa-14.0/4797/en/>`_
 * Mac OS X: `NI-VISA 14.0 <http://www.ni.com/download/ni-visa-14.0/4913/en/>`_
 
+Furthermore, the `Rohde & Schwarz VISA 5.12.1 <https://www.rohde-schwarz.com/us/applications/r-s-visa-application-note_56280-148812.html?rusprivacypolicy=0>`_ was tested with Linux as well. It is also available for all platforms.
```
Comment (caryan) [inline doc/installation.rst:21]: drop the `?rusprivacypolicy=0`

File: deps/build.jl [hunk 2]
```diff
@@ -1,7 +1,7 @@
 using BinDeps
 @BinDeps.setup
 
-visa = library_dependency("visa", aliases = ["visa64","VISA","/Library/Frameworks/VISA.framework/VISA"])
+visa = library_dependency("visa", aliases = ["visa64","VISA","/Library/Frameworks/VISA.framework/VISA", "librsvisa"])
```
Comment (caryan) [inline deps/build.jl:4]: :memo: leave a comment about what `libsrvisa` is

**Comment by casabre** [comment]
@caryan Supplemented your requests

**Comment by matthewware** [comment]
LGTM.  Looking forward to kicking the tires on this next week.  Thanks @casabre!

**Comment by catosp** [comment]
Hello guys! Is possible to run Instruments.jl on linux without installing NI VISA ? somenthing like pyvisapy but writen in pure Julia? NI VISA on linux works only for Centos, RedHat and Suse.

**Comment by casabre** [comment]
@catosp You can download the [Rohde & Schwarz VISA](https://www.rohde-schwarz.com/us/applications/r-s-visa-application-note_56280-148812.html) which is now also supported by the builder.

**Pull request merged**